multimedia/dvdauthor/Portfile

```diff
@@ -1,34 +1,42 @@
-PortSystem 1.0
-name		dvdauthor
-version		0.7.1
-revision	8
-categories	multimedia
-platforms	darwin
-maintainers	perry.ch:maurice
-license		GPL-2+
-
-description	A simple set of tools to help author a DVD.
-
-long_description	\
-	A simple set of tools to help you author a DVD. The idea is to be \
-	able to create menus, buttons, chapters, etc, but for now you can \
-	just take an mpeg stream (as created by mplex -f 8 from mjpegtools \
-	1.6.0) and write it to DVD.
-
-homepage	http://dvdauthor.sourceforge.net/
-master_sites	sourceforge:project/dvdauthor/dvdauthor/${version}
-
-checksums	rmd160  71bdee2048a45ea15b7d0e62e75675f9132006ca \
-		sha256  501fb11b09c6eb9c5a229dcb400bd81e408cc78d34eab6749970685023c51fe9
-
-depends_lib 	port:libdvdread \
-		port:libxml2 \
-		port:bzip2 \
-		port:zlib \
-		port:libpng \
-		port:libiconv \
-		port:freetype \
-		port:fontconfig \
-		port:ImageMagick
-
-worksrcdir	${name}
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dvdauthor
+version             0.7.2
+categories          multimedia
+platforms           darwin
+maintainers         perry.ch:maurice
```

Comment:
Do we have a github handle for this person?

Reply:
I am sorry, but I do not know.

Reply:
Some googling makes me wonder if it is @robbyn ?

Reply:
yep, that's me

Reply:
@robbyn, I added your GitHub handle as part of this pull request.
If you are fine with all of the changes, I can go ahead and merge it.

Reply:
Fine by me. You can merge it